src/components/NodeDisplay.tsx

```diff
@@ -0,0 +1,47 @@
+import { Html } from '@react-three/drei'
+import { FC, useMemo } from 'react'
+import * as THREE from 'three'
+
+interface NodeType {
+  positions: Float32Array
+}
+
+const NodeDisplay: FC<NodeType> = ({ positions }) => {
+  const indexedPositions = useMemo(() => {
```

Comment:
is it possible to check in unit tests that this indexedPositions has a correct value after element initialization?